### DISCUSSION TITLE
Support NEW Zephyrus G16 Ryzen AI 9 HX 370

### DISCUSSION THREAD

**Discussion by marv4u** [seerge/g-helper#2874] [Q&A]
Hey @all,

im looking forward to buy the new Zephyrus G16 with the Strix Ryzen AI 9 HX 370 CPU and im wondering about support.

Will it be supported like without any update, or do i have to wait for an updated release of g-helper, because its a totaly new cpu?

[https://rog.asus.com/de/laptops/rog-zephyrus/rog-zephyrus-g16-2024-ga605/spec/](url)

ah and before i forget... this tool is one of the reasons i went for the G16. Damn nice work, thanks for the ambisions and hard work!

**Comment by seerge** [comment]
@marv4u hi, most probably it should work out of the box, unless asus changes something specifically for that device

**Comment by marv4u** [comment]
looks like its not everything supported, i cant see controls for the lighting at the lid

anything else works as expected as far i can see

**Reply by seerge** [reply]
@marv4u try latest build posted in this thread 

https://github.com/seerge/g-helper/issues/2892

**Comment by BassemMohsen** [comment]
Just to to report, I have tested G Helper on AMD HX 370 G16.
It worked well out of the box, I have 0 complaints.

I used HWInfo to benchmark power before and after uninstalling ArmouryCrate and MyAsus.
It looks like Asus uses **0.6 Watt** in Idle mode to keep their background services running and functioning compared to G-Helper.

With G-Helper I'm able to hit 4.2Watt in Idle Silent mode, which is very nice.

Thanks a lot for this awesome app.

Slash lights, and Fn keys works well as well.

**Reply by seerge** [reply]
Good to know :)

**Reply by Ellettice** [reply]
How have you found battery life to be when configuring G-Helper to favor battery life (including eco mode for the gpu)? I was considering a p16 because it offers 64gb memory but wasn't trilled with battery life vs the zenbook s 16. I thought G-Helper settings could help get it closer but haven't seen many examples mentioning battery life with the HX 370/365 devices yet.

**Comment by BassemMohsen** [comment]
G Helper saves some minor battery. The ~600mw is not much in grand scheme of things.
The total Idle system discharge in silent mode with iGPU between **14.6 Watt** to **15.6 Watt** with screen 50% and keyboard backlight is off, and slash light medium.

The 600 mWatt savings are nice for less heat on the back and for fans to go to 0 RPM.

In mixed use, I would say you are looking for around 6 hours battery.
I think 365 might be better efficiency since it's lower core count, lower boost clocks.

Regarding the RAM, I think 32 GB is more than enough for my use cases when developing VSCode, WSL running and browsing the internet and doing some video gaming on the side.
But maybe you want 64GB for large video edits, or running own LLM locally?

**Reply by Ellettice** [reply]
64gb has been useful for development type work and data science type tasks. It's what I have in my 9720 xps. It had 32gb but it's been nicer ever since increasing it to 64gb. Battery life isn't good though only about 5 hours with light use (not even compiling) and it has some kind of issue where it thermally shutdowns due to severe overheating usually when idle and not interacting with it. 32gb is workable but it's a shame a couple years after getting the 9720, current 64gb options are considerably more expensive for less memory.

Was hoping for a 64gb hx 370/365 option with considerably better battery life (dedicated graphics not required and 16:10 or 3:2 screen) but basically everything is soldered memory and limited to 32gb max.

**Reply by BassemMohsen** [reply]
Dell XPS thin chassis with Intel 45Watt H chip, is not a good combo. I totally understand you.

I would recommend macbook m2/m3 if you want best battery and cooling temps. For 64 GB RAM, it's going to be pricey. Make sure your development tools are available natively on mac architecture first.

**Comment by BassemMohsen** [comment]
I can report a bug however, when I close the lid, the laptop takes 1 minute or so to go to sleep. During that minute, Fans comes loud.
This same behavior always happens whether it's Armory Crate and G Helper. I suspect it's a bug in how BIOS puts system in sleep mode.

**Reply by mergen3107** [reply]
This is actually Modern Standby (thanks, Microsoft). Laptops don’t go to S3 sleep anymore, but rather S0. Easy to check with `powercfg /a` command - check “S3 Suspend” item. 

Best alternative is Hibernate.
